src/main/java/frc/utils/applicationsutils/OutputFile.java

```diff
@@ -0,0 +1,97 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class OutputFile {
+
+    private static final File SELF_OUTPUT_FILE = getSelfOutputFile();
+
+    private final String name;
+    private final File file;
+
+    public OutputFile(String name) {
+        this.name = name;
+        this.file = DirectoryPathsConstants.OUTPUT_FILES_DIRECTORY_PATH.resolve(name).toFile();
+
+        ensureFolderExists();
+        ensureFileExists();
+        clear();
+    }
+
+    private void ensureFolderExists() {
+        File folder = DirectoryPathsConstants.OUTPUT_FILES_DIRECTORY_PATH.toFile();
+        if (!folder.exists()) {
+            folder.mkdir();
+        }
+    }
```

Comment:
Replace to Files functionality

Reply:
the create directory function of the Files class requires me to use a try catch. it is a lot cleaner to use `folder.mkdir()`

Reply:
But it works safer and better